qctrlopencontrols/driven_controls/predefined.py

```diff
@@ -104,26 +109,28 @@ def new_predefined_driven_control(scheme=PRIMITIVE, **kwargs):
     return driven_control
 
 
-def _predefined_common_attributes(maximum_rabi_rate, rabi_rotation, azimuthal_angle):
+def _predefined_common_attributes(
+    azimuthal_angle: float, rabi_rotation: float, maximum_rabi_rate: float = 2 * np.pi,
+) -> Tuple[float, float, float]:
     """
     Adds some checks etc for all the predefined pulses
 
     Parameters
     ----------
+    azimuthal_angle : float
+        The azimuthal position of the pulse.
     rabi_rotation : float
         The total polar angle to be performed by the pulse.
-        Defined in polar coordinates.
-    maximum_rabi_rate : float
+        Defined in polar coordinates. Defaults to None.
```

Comment:
Remove the comment about defaulting to None

Reply:
done